lib/index.js

```diff
@@ -596,11 +596,17 @@ export function _checkCredential(credential) {
     }
   });
 
-  // check expires is a date
-  if('expirationDate' in credential &&
-      !dateRegex.test(credential.expirationDate)) {
-    throw new Error(
-      `"expirationDate" must be a valid date: ${credential.expirationDate}`);
+  if('expirationDate' in credential) {
+    // check expires is a date
+    if(!dateRegex.test(credential.expirationDate)) {
+      throw new Error(
+        `"expirationDate" must be a valid date: ${credential.expirationDate}`);
+    }
+    // check if credential has expired
+    const now = new Date().toISOString();
```

Comment:
`now` should be a option that defaults to the current date time. Additionally, we should be checking that `now` is on or after `issuanceDate`.